### DISCUSSION TITLE
MPD player in Music Assistant Beta version and then next stable versions

### DISCUSSION THREAD

**Discussion by alexey2626** [music-assistant/support#1170] [Feature requests and ideas]
The latest stable version has an MPD player, but the Beta version (2.0.0b22) does not.
Can you please bring back the MPD?

**Comment by chrisghu** [comment]
I also would like to have such a feature. With the previous version there was no issue to use the mpd players of home assisant but with the latest beta v2 they are not even discovered.

**Comment by Stumpi73** [comment]
This feature is the most importend feature for me - i have no other players in my network available.

**Reply by OzGav** [reply]
Make sure you have voted in the player provider poll!

**Reply by AlterDepp** [reply]
Where is the player provider poll?

**Reply by OzGav** [reply]
Look at the BIG buttons at the top of the discussions

**Comment by melwinek** [comment]
For me, mpd support would also be very useful.

**Comment by melwinek** [comment]
@OzGav Is there a chance for mpd in the near future?

**Reply by OzGav** [reply]
There is no dev working in this so not in the near future. Only 7 upvotes for this. For my info, as I don’t use MPD, what does this do that MA doesn’t…it seems like an unusual fit…

**Reply by AlterDepp** [reply]
MPD is on open source music backend. Home Assistant Media player streams music to MPD, which is installed on a raspberry pi, which is connected to my active speakers. We would like to do the same with Music Assistant.

**Reply by OzGav** [reply]
Based on what you have written I don’t see why you need MPD. Just install picoreplayer on the Raspberry Pi and MA will connect directly to that.

**Reply by AlterDepp** [reply]
Hm, I did not find a package, to install picoreplayer, only a complete image. So I installed squeezelite on my raspi. Worked out ouf the box, without any configuration. I also installed the squeezelite add-on from https://github.com/pssc/ha-addon-squeezelite on my home assistant yellow. Worked out of the box, too. For both systems, music assistant found the players as Slimproto players immediately after installing. Would be nice to find this in any documentation more easy.

**Reply by OzGav** [reply]
Well the FAQ does have a section called “How to stream to my local HA device”

**Reply by Bram-diederik** [reply]
Well I'm investigating to migrate my multi room mpd/snapcast system to mass. and I also use it for music/radio selection on my phone. using a http stream over vpn to vlc on my android phone. 

mass cant do that yet. mpd has tons of integration possibilities.

**Reply by OzGav** [reply]
Passing comment on Discord suggested someone has squeezelite running on Android ink using VPN support so you could maybe look into that?

**Reply by Bram-diederik** [reply]
Yeah. But with vlc and a http stream i can make a playlist that can be edited and read by vlc.

When i want to play a radio station the playlist will contain the radio station so that i dont have my home unnecessary in between.

**Reply by OzGav** [reply]
Editing playlists is due on MA 2.1 Is there anything else that you need to only use MA?

**Comment by melwinek** [comment]
Then what do you recommend on raspberries as satelitte players? Squeezelite?

**Reply by OzGav** [reply]
Yes picoreplayer

**Comment by OzGav** [comment]
This will be added in 2.9.0beta7

**Comment by OzGav** [comment]
The feature request has been implemented successfully. As such this feature request will be closed and locked. If there are enhancements desired then open a new [feature request](https://github.com/orgs/music-assistant/discussions/categories/feature-requests-and-ideas). If there is a question about how this works then check the [documentation](https://beta.music-assistant.io/), or ask a question in the [Q&A](https://github.com/orgs/music-assistant/discussions/categories/q-a) area or on Discord. If there are problems then ask a question in the [Q&A](https://github.com/orgs/music-assistant/discussions/categories/q-a) area or on Discord or open an [issue](https://github.com/music-assistant/support/issues). Thanks for your interest in seeing this idea being brought to life.
